deployment/deploy.sh

```diff
@@ -19,6 +19,10 @@ cd ..
 # Copy the constants in the build folder
 cp $PATH_TO_CONSTANTS_FILE public/assets/firebase_constants.json
 
+# Copy the latest commit sha1 hash on origin/master into the build folder
+LASTEST_COMMIT_HASH=$(git rev-parse origin/master)
```

Comment:
I think it's possible (and likely) that the deploy script will be run from other places in the git tree rather than master. Is there a way of getting the current commit hash rather than the latest on master?

Reply:
Yes, there is a way I think. Lemme refactor the code with the change

Reply:
Check the new changes. I've refactored it to get the hash from the current branch